proposals/cloudedgeclientCG/charter.md

```diff
@@ -35,7 +35,7 @@ The Group will have succeeded if it can achieve the following:
 ## Deliverables
 
 ### Specifications
-No Specifications will be produced under the current charter.
+* Specifications (API, protocol design etc.) will be produced under the current charter.
```

Comment:
Perhaps change "will" to "may" as Specifications can be one of the possible deliverables for CG to produce. It doesnt have to be a must.

Reply:
I'd remove the `(API, protocol design etc.)` part here. It could be  laid out more elaborately in the Deliverables section, see for instance the [Web Bluetooth charter](https://www.w3.org/community/web-bluetooth/web-bluetooth-community-group-charter/), or [Solid CG charter](https://www.w3.org/community/solid/charter/). The latter is a more recent CG and it's in a way similar to this, so we could take that as a basis and trim it to our purpose.

Reply:
Yes. I think we can remove '(API, protocol design etc.)" and elaborate a bit the possible documents or work items.

Reply:
I suggest "The Community Group may develop specifications within the Scope of this Charter". We can update the charter in future once the CG starts and actual specifications start to be identified.